src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -37,16 +37,21 @@ interface BuildLocator {
      * By default only successful builds are returned, call this method to include failed builds as well.
      */
     fun withAnyStatus() : BuildLocator
-
     fun withStatus(status: BuildStatus): BuildLocator
+
     fun withTag(tag: String): BuildLocator
 
-    fun withBranch(branch: String): BuildLocator
+    /**
+     * By default failed to start builds are filtered out, call this method to include them.
+     */
+    fun withAnyFailedToStart(): BuildLocator
```

Comment:
I'm afraid this will make the option less discoverable and I'm also not sure if there mightn't be a successful build that failed to start. Say, if status had been changed manually.